libs/blocks/locui-create/store.js

```diff
@@ -9,8 +9,20 @@ import {
   getMilocUrl,
 } from './utils/utils.js';
 
+const testInfo = {
```

Comment:
Not sure whether we need this block as part of PR? I suppose this must be added to do local testing.

Reply:
removed